local-running/src/main/java/uk/gov/di/ipv/coreback/CoreBack.java

```diff
@@ -29,15 +26,17 @@ public CoreBack() throws IOException, URISyntaxException {
         app.get("/audit-events", auditHandler::getAuditEvents);
 
         // Internal APIs
-        app.post("/session/initialise", lambdaHandler::initialiseSession);
-        app.post("/journey/{event}", journeyEngineHandler::journeyEngine);
-        app.post("/cri/callback", lambdaHandler::criCallback);
-        app.get("/user/proven-identity-details", lambdaHandler::getProvenUserIdentityDetails);
+        app.post("/session/initialise", ctx -> new LambdaHandler().initialiseSession(ctx));
+        app.post("/journey/{event}", ctx -> new JourneyEngineHandler().journeyEngine(ctx));
```

Comment:
This change has a significant impact on the performance of the API tests, so I will attempt to use ThreadLocal to ensure the ConfigService is updated & referenced within the scope of each process thread